qCC/ccCommandLineCommands.cpp

```diff
@@ -1176,44 +1176,49 @@ bool CommandOctreeNormal::process(ccCommandLineInterface& cmd)
 		}
 
 		bool success = false;
+		bool normalsAlreadyOriented = false;
 		if (useGridStructure)
 		{
-			cmd.print(QObject::tr("computeNormalsWithGrids started..."));
-			success = cloud->computeNormalsWithGrids(angle, progressDialog.data());
-			if(success)
+			if (orientation == ccNormalVectors::UNDEFINED)
 			{
-				cmd.print(QObject::tr("computeNormalsWithGrids success"));
+				ccLog::Print("\tcompute + orient normals with grids");
```

Comment:
Not sure to understand why you use all these tabs? (\t)

Reply:
It is just to format the messages for a better reading.
Maybe not a good idea?